File: app/src/main/java/com/botpa/turbophotos/sync/SyncEvent.java
package com.botpa.turbophotos.sync;

public class SyncEvent {

    public final String command;
    public final String stringValue;
    public final int intValue;

    public SyncEvent(String command, String value) {
        this.command = command;
        this.stringValue = value;
        this.intValue = 0;
    }

    public SyncEvent(String command, int value) {
        this.command = command;
        this.stringValue = null;
        this.intValue = value;
    }

}

File: app/src/main/java/com/botpa/turbophotos/sync/SyncEvent.kt
package com.botpa.turbophotos.sync

class SyncEvent {

    @JvmField val command: String
    @JvmField val valueString: String
    @JvmField val valueInt: Int

    constructor(command: String, value: String) {
        this.command = command
        this.valueString = value
        this.valueInt = 0
    }

    constructor(command: String, value: Int) {
        this.command = command
        this.valueString = ""
        this.valueInt = value
    }

}